near-sdk/tests/store_performance_tests.rs

```diff
@@ -422,3 +434,49 @@ async fn iterable_vs_unordered() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_lazy() -> anyhow::Result<()> {
+    let (account, contract_id) = setup(Contract::LazyContract).await?;
+
+    let res = account
+        .call(&contract_id, "insert_delete")
+        .args_json((700,))
+        .max_gas()
+        .transact()
+        .await?
+        .unwrap();
+
+    perform_asserts(res.total_gas_burnt.as_gas(), &Collection::LazyOption);
+
+    let res = account
+        .call(&contract_id, "insert_delete_flush_once")
+        .args_json((1700,))
+        .max_gas()
+        .transact()
+        .await?
+        .unwrap();
+
+    perform_asserts(res.total_gas_burnt.as_gas(), &Collection::LazyOption);
+
+    let res = account
+        .call(&contract_id, "flush")
+        .args_json((2400000,))
+        .max_gas()
+        .transact()
+        .await?
+        .unwrap();
+
+    perform_asserts(res.total_gas_burnt.as_gas(), &Collection::LazyOption);
+
+    let res = account
+        .call(&contract_id, "insert_flush")
+        .args_json((1200,))
+        .max_gas()
+        .transact()
+        .await?
+        .unwrap();
+
+    perform_asserts(res.total_gas_burnt.as_gas(), &Collection::LazyOption);
+    Ok(())
```

Comment:
Should we also add `get` estimation? As we do for `contains`
Also maybe `insert_take`